classrooms/views.py

```
@@ -2,2 +1,124 @@
+@login_required
+@require_http_methods(["POST", "GET"])
+def create_classroom(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        form = ClassroomForm(request.POST, request.FILES)
+        if form.is_valid():
+            saved_classroom = form.save()
+            if request.user:
+                ClassroomTeacher.objects.create(classroom=saved_classroom, teacher=request.user)
+            messages.success(request, "Classroom saved successfully!")
+            return redirect(reverse("classroom_details", kwargs={"slug": saved_classroom.slug}))
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)